utils/misc.ts

```diff
@@ -24,3 +24,12 @@ export function readImageType(buffer: ArrayBuffer) {
   if (buffer) return imageType(Buffer.from(buffer));
   return null;
 }
+
+export function catchAxiosError(promise: Promise<any>) {
```

Comment:
We do know the promise should be of axios http request type though?

Reply:
Since [axios.get](https://github.com/likecoin/app-like-co/pull/319/files#diff-a44f2b879ce5cfedfee1de745a1ecc3eca574e5a45d4ae1040c80b846874c643R134-R135) itself returns an AxiosPromise, the catchAxiosError function is designed to handle the returned promise.

Reply:
Just type the param `promise` as `AxiosPromise` then?